### PR TITLE
Added a property called "rotate"

### PR DESCRIPTION
**Pull request by diotto**
I've created a property called rotate and changed example to use it. In example i changed the behavior of words to stay only in 0 ou 90 degrees.

### CODE REVIEW

**Comment by ScottChapman** [comment]
Would love this, please merge?

**Comment by diotto** [comment]
@ScottChapman as my pull request was not accepted yet, you can see my modified version at https://github.com/diotto/d3-wordcloud

File: d3.wordcloud.js [hunk 1]
```diff
@@ -49,6 +50,12 @@
         return wordcloud;
     }
 
+    wordcloud.rotate = function(x) {
+      if (!arguments.length) return rotate;
+      rotate = x == null ? function() {return (~~(Math.random() * 6) - 3) * 30;} : x;
+      return wordcloud
```
Comment (wvengen) [inline d3.wordcloud.js:56]: missing semicolon

File: d3.wordcloud.js [hunk 2]
```diff
@@ -63,6 +70,7 @@
         .text(function(d) {
           return d.text;
         })
+        .rotate(rotate)
```
Comment (wvengen) [inline d3.wordcloud.js:73]: Why not put the default function here, and allow `rotate` to be null? Avoids having the function twice.

**Review by wvengen — CHANGES_REQUESTED** [review]
Thanks for the contribution. There are some small comments, would love to integrate this when ready!

**Pull request closed without merging**